MdePkg/Library/BasePanicLib/PanicLib.c

```diff
@@ -0,0 +1,59 @@
+/** @file
+  Base Panic Library.
+
+  Copyright (c) Microsoft Corporation.
+

```

Comment:
Need SPDX identifier.

```suggestion
  SPDX-License-Identifier: BSD-2-Clause-Patent

```

Reply:
Fixed.